### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_HighContrast:'Graphs' present on the page don't inherit the 'High Contrast' Mode.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4004]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10

**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4. Apply HighContrast(Black) mode.
5.Verify whether the graphs inherit the HighContrast mode or not.

**Actual**
Graphs present on the page don't inherit the 'HighContrast' mode settings.

**Expected**
Graphs should also inherit the HighContrast mode settings, so that low vision user
would be able to understand the graphs clearly.

**User Impact**
Low vision user would not be able to see the graphs and won't be able to understand the graphs clearly.
**Attachment:**
![a11y_azurenotebooks_jupyter_creationanddeployment_highcontrast](https://user-images.githubusercontent.com/39204909/45816876-9015e500-bcfa-11e8-9747-3cef93e92d1a.png)
